build/buildBuildImages.sh

```diff
@@ -219,6 +219,20 @@ buildDockerImage "$BUILD_IMAGES_DOCKERFILE" \
 				"$ORYXTESTS_BUILDIMAGE_REPO" \
 				"$DEVBOX_BUILD_IMAGES_REPO"
 
+echo
+echo "-------------Creating VSO build image-------------------"
+builtImageName="$ACR_BUILD_VSO_IMAGE_NAME"
+docker build -t $builtImageName \
+	--build-arg AGENTBUILD=$BUILD_SIGNED \
+	--build-arg AI_KEY=$APPLICATION_INSIGHTS_INSTRUMENTATION_KEY \
+	$storageArgs \
+	$buildMetadataArgs \
+	-f "$BUILD_IMAGES_VSO_DOCKERFILE" \
+	.
+echo
+echo "$builtImageName" >> $ACR_BUILD_IMAGES_ARTIFACTS_FILE
+createImageNameWithReleaseTag $builtImageName
+
```

Comment:
nit: split up multiple build image creation from current buildbuildimage .. <im doing it>

Reply:
> nit: split up multiple build image creation from current buildbuildimage ..

im working on this btw

Reply:
> im working on this btw

So I am going to ignore this then.